lib/src/crypto/src/cipher_params.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:ably_flutter/ably_flutter.dart';
 
-/// params to configure encryption for a channel
+/// Params to configure encryption for a channel
```

Comment:
Please add a dot a the end of the sentence.

Reply:
Done in 6c4b6b04df275b6136936cfec3d2b520b0477a55